src/Layout/Footer/Footer.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+/** Services */
+import { skipCurrentSong } from '@services/api';
 /** Utils */
 /** Components */
 import ProgressBar from './ProgressBar';
 import NowPlaying from './NowPlaying';
 /** Styled components */
 import { FooterContainer, FooterWrapper } from './footer.styled';
 
+import './SkipButton.scss';
+
 const Footer = () => {
   const current = useSelector(state => state.playing.current);
 
+  const skipHandler = async () => {
+    if (current && current.publicId) {
+      const roomSessionModel = { roomSessionId: current.publicId };
```

Comment:
Something seems off here for me although I'm not sure on what the endpoint expects to receive. We're sending an object with a roomSessionId property but assigning the current song ID

@pabravil can you provide more information?

Reply:
You're right, should provide the current session ID, yet it's still working.

Reply:
For the moment I'll leave it without parameters and we'll see later how to handle session ID.